src/Mvc/test/WebSites/BasicWebSite/StartupWithCustomInvalidModelStateFactory.cs

```diff
@@ -24,15 +24,11 @@ public void ConfigureServices(IServiceCollection services)
 
             services.Configure<ApiBehaviorOptions>(options =>
             {
-                var previous = options.InvalidModelStateResponseFactory;
                 options.InvalidModelStateResponseFactory = context =>
                 {
-                    var result = (BadRequestObjectResult)previous(context);
```

Comment:
This test was really janky. This broke because it expected a vanilla dictionary but now gets a ProblemDetails. We wouldn't recommend users to write something like this.